nautobot/extras/urls.py

```diff
@@ -396,4 +397,23 @@
         name="computedfield_changelog",
         kwargs={"model": ComputedField},
     ),
+    # Custom fields
+    path("custom-fields/", views.CustomFieldListView.as_view(), name="customfield_list"),
+    path("custom-fields/add/", views.CustomFieldEditView.as_view(), name="customfield_add"),
+    path(
+        "custom-fields/delete/",
+        views.CustomFieldBulkDeleteView.as_view(),
+        name="customfield_bulk_delete",
+    ),
+    path("custom-fields/<uuid:pk>/", views.CustomFieldView.as_view(), name="customfield"),
+    path(
+        "custom-fields/<uuid:pk>/edit/",
+        views.CustomFieldEditView.as_view(),
+        name="customfield_edit",
+    ),
+    path(
+        "custom-fields/<uuid:pk>/delete/",
+        views.CustomFieldDeleteView.as_view(),
+        name="customfield_delete",
+    ),
```

Comment:
Can we use the `slug` (`name`, for now) of the custom-field instead of the UUID for these URLs?